admin_manual/configuration_server/harden_server.rst

```diff
@@ -12,6 +12,23 @@ is based on the assumption that you run ownCloud Server on Apache2 on a Linux en
           however in some cases that are considered second level as defense administrators are encouraged to check these
           hardenings manually.
 
```

Comment:
This is a bit awkward; how about "ownCloud will warn you in the administration interface if some critical security-relevant options are missing. However, it is still up to the server administrator to review and maintain system security."